bentoml/_internal/configuration/containers.py

```diff
@@ -315,8 +300,8 @@ def metrics_client(
         )
 
     # Mapping from runner name to RunnerApp file descriptor
-    remote_runner_mapping = t.cast(Provider[t.Dict[str, int]], providers.Static(dict()))
-    plasma_db: "PlasmaClient" = providers.Static(None)  # type: ignore[reportUnknownVariableType]
+    remote_runner_mapping = providers.Static[t.Dict[str, int]](dict())
+    plasma_db = providers.Placeholder["PlasmaClient"]()  # type: ignore
```

Comment:
Any reason this isn't just `t.Optional` and `None`?

Reply:
I think for some of the configs, `None` actually means something, whereas `Placeholder` means user's config yaml did not touch this field

Reply:
but in this case, I think `t.Optional` and `None` is better. @bojiang any reason we are not doing that?

Reply:
yeah Optional makes sense here. will modify it.